service/src/main/java/com/fecklessweasel/service/datatier/CourseTable.java

```diff
@@ -17,6 +17,7 @@
 public class CourseTable {
 
     private static String INSERT_ROW = "insert into Course (univid, deptid, courseNumber) values (?,?,?)";
+    private static String LOOKUP_WITH_DEPT = "SELECT * FROM Course WHERE Course.deptid=?";
```

Comment:
What if there are 10's of thousands of schools? (Thousands of schools times hundreds of courses). We should support paging results. Also, results should be sorted.